### PR TITLE
[11.0][mig] account mass reconcile by purchase line

### PR DESCRIPTION
**Pull request by JordiBForgeFlow**
Standard migration to 11.0

### CODE REVIEW

**Comment by MiquelRForgeFlow** [comment]
@jbeficent could you check on runbot? :)

**Review by MiquelRForgeFlow — APPROVED** [review]

**Comment by JordiBForgeFlow** [comment]
@aheficent can you review?

File: account_mass_reconcile_by_purchase_line/models/mass_reconcile.py [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+
+from odoo import models
+
+
+class AccountMassReconcileMethod(models.Model):
+    _inherit = 'account.mass.reconcile.method'
+
+    @staticmethod
+    def _get_reconcilation_methods():
+        methods = super()._get_reconcilation_methods()
```
Comment (AaronHForgeFlow) [inline account_mass_reconcile_by_purchase_line/models/mass_reconcile.py]: File "/home/odoo/build/OCA/bank-statement-reconcile/account_mass_reconcile/models/mass_reconcile.py", line 85, in _selection_name
    return self._get_reconcilation_methods()
  File "/home/odoo/build/OCA/bank-statement-reconcile/account_mass_reconcile_by_purchase_line/models/mass_reconcile.py", line 12, in _get_reconcilation_methods
    methods = super()._get_reconcilation_methods()
RuntimeError: super(): no arguments

**Review by AaronHForgeFlow — COMMENTED** [review]
Got an error  here when clicking to create a mass reconcile record on runbot.

File: account_mass_reconcile_by_purchase_line/models/mass_reconcile.py (hunk 1 above)
Comment (MiquelRForgeFlow) [reply account_mass_reconcile_by_purchase_line/models/mass_reconcile.py]: Fixed. Could you try again, @aheficent ?

Comment (AaronHForgeFlow) [reply account_mass_reconcile_by_purchase_line/models/mass_reconcile.py]: what's wrong with runbot? I forced rebuild twice but it's never available.

**Comment by JordiBForgeFlow** [comment]
@aheficent not sure what's wrong with runbot, but it's not related to this PR.

**Review by AaronHForgeFlow — CHANGES_REQUESTED** [review]
Still errored

**Review by AaronHForgeFlow — APPROVED** [review]
:+1: Functional tested

**Pull request merged**